remote-execution.md

```diff
@@ -200,13 +223,22 @@ the context menu.
 You even have the option of disabling auto-copying of files in the project to
 the server if you want to manually control the copying process yourself. 
 
+If you invoke an R command through an explicit Visual Studio command, we will
+ensure that files get copied to the remote machine. For example, right-clicking
+on a file in Solution Explorer and running the Source Selected File(s) command
+will copy the files to the remote workspace before sourcing them. Similarly,
+when you press F5 to debug your project; RTVS will copy files to the remote
+workspace before attaching the debugger and running your program. Since we
+cannot reliably intercept all R functions, calling functions such as `source()`
+or `runApp()` (for Shiny applications) from the Interactive Window will **not**
+copy any files from your computer to the remote workspace.
+
 ### The long term goal of Local and Remote Workspaces
```

Comment:
We may want to document `rtvs::fetch_file`

![image](https://cloud.githubusercontent.com/assets/12820357/23912030/cccd2b94-089b-11e7-9cf1-cf93e3673fc3.png)